gensim/similarities/docsim.py

```diff
@@ -154,39 +179,60 @@ class Similarity(interfaces.SimilarityABC):
     def __init__(self, output_prefix, corpus, num_features, num_best=None, chunksize=256, shardsize=32768, norm='l2'):
         """
         Construct the index from `corpus`. The index can be later extended by calling
-        the `add_documents` method. **Note**: documents are split (internally, transparently)
-        into shards of `shardsize` documents each, converted to a matrix, for faster BLAS calls.
-        Each shard is stored to disk under `output_prefix.shard_number` (=you need write
-        access to that location). If you don't specify an output prefix, a random
-        filename in temp will be used.
-
-        `shardsize` should be chosen so that a `shardsize x chunksize` matrix of floats
-        fits comfortably into main memory.
+        the `add_documents` method.
 
-        `num_features` is the number of features in the `corpus` (e.g. size of the
-        dictionary, or the number of latent topics for latent semantic models).
-
-        `norm` is the user-chosen normalization to use. Accepted values are: 'l1' and 'l2'.
-
-        If `num_best` is left unspecified, similarity queries will return a full
-        vector with one float for every document in the index:
+        Parameters
+        ----------
+        output_prefix : str
+            Prefix for shard. If you don't specify an output prefix, a random filename in temp will be used.
+        corpus : iterable of iterable of (int, int)
+            Corpus in BoW format.
+        num_features : int
+            Size of the dictionary.
+        num_best : int, optional
+            If set, return only the `num_best` most similar documents, always leaving out documents with similarity = 0.
+            Otherwise, return a full vector with one float for every document in the index.
+        chunksize : int, optional
+            Size of block.
+        shardsize : int, optional
+            Size of shards should be chosen so that a `shardsize x chunksize` matrix of floats fits comfortably into
+            main memory.
+        norm : str, optional
+            Normalization to use. Accepted values: {l1, l2}.
```

Comment:
In this case, instead of 
```
norm : str, optional
    Normalization to use. Accepted values: {l1, l2}.
```

should be
```
norm : {'l1', 'l2'}, optional
    Normalization to use.
```

Reply:
This is better notation when we have several string pre-defined values.